### ISSUE TITLE
Check config file does not work with register

### ISSUE THREAD

**Issue by SeyZ** [SeyZ/baboon#44]
I have the following configuration file:

```
[server]
max_stanza_size = 65535
pubsub = pubsub.baboon-project.org
streamer = streamer.baboon-project.org
server = admin@baboon-project.org/baboond

[user]
passwd = secret
jid = seyz@baboon-project.org

[cortex-web]
scm = git
path = /home/seyz/workspace/internal/cortex-web

# Example of project definition
#[awesome_project]      # The project name on the baboon server
#path = /pathto/project # The project path of your system
#scm = git              # The source code manager you use for this project
#enable = 1             # You want baboon to actually watch this project
```

I replaced 'server' entry by 'master' for testing and:

```
(baboon)seyz@dogwood ~/workspace/external/baboon (master) $ python bin/baboon register marci --nosave
Password: 
Confirm password: 

Registration in progress...
Traceback (most recent call last):
  File "bin/baboon", line 9, in <module>
    Main()
  File "/home/seyz/workspace/external/baboon/baboon/main.py", line 29, in __init__
    getattr(commands, self.which)()
  File "/home/seyz/workspace/external/baboon/baboon/commands/commands.py", line 43, in register
    transport = RegisterTransport(callback=_on_action_finished)
  File "/home/seyz/workspace/external/baboon/baboon/transport.py", line 426, in __init__
    super(RegisterTransport, self).__init__()
  File "/home/seyz/workspace/external/baboon/baboon/transport.py", line 45, in __init__
    self.server_addr = config['server']['master']
KeyError: 'master'
```

**Comment by SeyZ** [comment]
WIP, you can continue (I have to go :p)

**Comment by SeyZ** [comment]
Fixed with the commit 63a067dd831258f1642c5668258d7dbc55dbfdc4
